contents/blog/how-to-treat-your-pricing-like-a-product.md

```diff
@@ -0,0 +1,102 @@
+---
+date: 2023-04-09
+title: How to treat your pricing like a product (and why it matters)
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - raquel-m-smith
+featuredImage: ../images/blog/happy-hog.png
+featuredImageType: full
+category: Product growth
+tags:
+  - Product engineers
+  - Guides
+
+---
+"Pricing is important" is common knowledge – price determines profit margin, which directly or indirectly determines nearly everything else. 
+
+What isn't common knowledge is twofold:
```

Comment:
```suggestion
"Pricing is everything" might not be 100% true, but it is close – price determines profit margin, which directly or indirectly determines nearly everything else.

There is a lot of common knowledge about pricing, but what isn't is twofold:
```
An alternative idea for an intro. "Pricing is important" feel a bit weak.

Reply:
Original isn't perfect, but alternative feels a bit wordy. Leaving as-is.